index.html

```diff
@@ -3951,41 +3951,191 @@ <h2>Syntaxes</h2>
         <h3>JSON-LD</h3>
 
         <p>
+The world of credentials is vast and varied. A physical wallet places
+relatively few constraints on the shape and size of those credentials–and it
+places no restriction on their contents. Likewise, Verifiable Credential Data
+Model’s use of [[!JSON-LD]] provides the consistent shape for all compliant
+Verifiable Credentials, so that digital wallets can store credentials with
+widely varying content, providing value to a plethora of communities, and
+enabling many diverse use cases–from coupons to authentication.
+        </p>
+
+        <p>
+Consequently, Wallet providers can have the confidence that any Verifiable
+Credential will “fit” while Issuers and Verifiers benefit from a common
+exchange representation that provides built in extensibility without the
+high costs from centralized terminology registration or JSON format churn.
+        </p>
+
+        <p>
+Issuers can add claims terminology unique to their communities via [[!JSON-LD]]’s
+context files rather than be dependent on a single, centralized registry
+system. This webby decentralized approach provides infinite scalability. Future
+standardization efforts, then focus on vocabulary definitions and
+object-oriented modeling provided by communities of practice vs. idiosyncratic
+JSON format creation for individual use cases or scenarios.
```

Comment:
```suggestion
Issuers can add claim terminology unique to their communities via [[!JSON-LD]]’s
context files rather than being dependent on a single, centralized registry
system. This webby, decentralized approach provides infinite scalability. Future
standardization efforts may then focus on vocabulary definitions and
object-oriented modeling provided by communities of practice rather than idiosyncratic
JSON format creation for individual use cases or scenarios.
```